src/components/CIPipelineN/YAMLScriptComponent.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react'
+import { ScriptType } from '../ciPipeline/types'
+import CodeEditor from '../CodeEditor/CodeEditor'
+import ReactSelect from 'react-select'
+
+export function YAMLScriptComponent({
+    editorValue,
+    handleEditorValueChange,
+    showSample,
+    height,
+}: {
+    editorValue: string
+    handleEditorValueChange: (string) => void
+    showSample?: boolean
+    height?: string
+}) {
+    const scriptTypeOptions: { label: string; value: string }[] = [ScriptType.SHELL, ScriptType.CONTAINERIMAGE].map(
+        (scriptType) => ({ label: scriptType, value: scriptType }),
+    )
+    const [selectedScriptType, setSelectedScriptType] = useState<{ label: string; value: string }>(scriptTypeOptions[0])
+    const [selectedTab, setSelectedTab] = useState<string>('editor')
+
+    const tempMultiSelectStyles = {
```

Comment:
Rename it to customMultiSelectStyles instead of temp. Also, if possible keep this in utils :)